src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/SignatureHelp/SignatureHelpEndpoint.cs

```diff
@@ -11,7 +12,7 @@
 using Microsoft.CodeAnalysis.Razor.Protocol;
 using Microsoft.CodeAnalysis.Razor.Workspaces;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
-using LS = Microsoft.VisualStudio.LanguageServer.Protocol;
+using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
```

Comment:
>LSP [](http://example.com/codeflow?start=6&length=3)

Another super-nit :smile: - seems like we just use SignatureHelp from that namespace most of the time, I'd probably alias that class as LspSignatureHelp (also I'd use Lsp to match other classes, though we don't seem super-consistent). Same comment applies to a couple of other classes in this PR.

Reply:
100% agreed, I much prefer that method too, I was just blindly fixing a typo here :)

Will update.